### DISCUSSION TITLE
Can I get the include field id inner in include?

### DISCUSSION THREAD

**Discussion by chioio** [prisma/prisma#12900] [Q&A]
### The code first:

```ts
const includes: Prisma.CreationInclude = {
      owner: {
        select: {
          id: true,
          followers: {
            select: {
              id: true,
            },
            where: {
              followee: {
                followee_id: {
                  equals:  # here I want get the owner field id.
               },
            },
         }
      },
   },
}
```

Can someone tip me, please! Thanks!

**Comment by nurul3101** [comment]
Hey @chioio 👋 
If I understand this correctly, I don't think that is possible, to confirm what you want to do is pass the `id` that you got by the `select` clause from the `owner` relation, right?

```
const includes: Prisma.CreationInclude = {
      owner: {
        select: {
          id: true, <- You want to pass this id in the below equals clause, correct?
          followers: {
            select: {
              id: true,
            },
            where: {
              followee: {
                followee_id: {
                  equals:  # here I want get the owner field id.
               },
            },
         }
      },
   },
}
```
This will be in a find query, so won't you have access to the search field which could be the owner id beforehand?
If this is not what you want to do, sharing schema file and the query which you are invoking would be helpful.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
